src/static/js/pad_impexp.js

```diff
@@ -243,8 +243,19 @@ var padimpexp = (function()
       if (status !== "ok")
       {
         importFailed(status);
+      }else{
+        $('#import_export').removeClass('popup-show');
+      }
+
+      // directDatabaseAccess is a string not a boolean
+      if(directDatabaseAccess !== 'undefined'){
```

Comment:
Do we really want to do this instead of finding the server side bug?

Reply:
It was doing this before....  This isn't new code, it's just fixing it up.

Reply:
Look at this. May it be here?

https://github.com/ether/etherpad-lite/blob/789dd7acd6a92d4b750ba2bf290690090052ecb3/src/node/handler/ImportHandler.js#L283